content/workflows/fairness-equal-odds-postprocessing.md

```diff
@@ -0,0 +1,12 @@
++++
+title= "Equal Odds Postprocessing"
+images =  ["/workflows/images/fairness-equal-odds-postprocessing.png"]
+type = "workflows"
+blog_link =  "/blog/2023/2023-08-30-fairness-equal-odds-postprocessing/"
+video = ""
+download = "fairness-equal-odds-postprocessing.ows"
+workflows = ["Fairness"]
+weight = 1004
++++
+
+Another way to mitigate bias besides the ones we have shown before is to use a postprocessing algorithm on the model's predictions. This workflow illustrates using the Equal Odds widget as a post-processor for the Logistic Regression model. To use the post-processor, we need to connect any model to the Equalized Odds Postprocessing widget along with any needed pre-processors. Doing so ensures our model's predictions get post-processed before we evaluate them. We then connect the Equalized Odds Postprocessing widget to the Test and Score widget and visualize the results using a Mosaic Display.
```

Comment:
"besides the ones we have shown before" - workflows should work by themselvers, do not reference others

Reply:
For me, everything after "Doing so" is fairly obvious, non-specific to this use-case, and should thus be removed.